packages/twenty-website/src/content/developers/local-setup.mdx

```diff
@@ -262,8 +279,6 @@ This should work out of the box with the eslint extension installed. If this doe
 }
 ```
 
-#### Docker container build
-
 To successfully build Docker images, ensure that your system has a minimum of 2GB of memory available. For users of Docker Desktop, please verify that you've allocated sufficient resources to Docker within the application's settings.
```

Comment:
we should still remove this paragraph

Reply:
@BOHEUS

Reply:
@charlesBochet but I've removed it earlier and restored as per this [comment](https://github.com/twentyhq/twenty/pull/7655#discussion_r1798472220), so... delete it or not? 😂

Reply:
Delete the title + paragraph and keep the <ArticleEditContent></ArticleEditContent> :p

Reply:
Done!